src/Microsoft.Fast.Components.FluentUI/Components/Base/InputHelpers.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System.Globalization;
+
+namespace Microsoft.Fast.Components.FluentUI;
+
+internal static class InputHelpers<TValue>
+{
+
+    public static string GetMaxValue()
```

Comment:
Even if the class is `internal`, it's always better to put the methods in `internal` too (it's clearer).

Reply:
Done!